onboard/catkin_ws/src/task_planning/scripts/move_tasks.py

```diff
@@ -4,35 +4,73 @@
 from tf.transformations import quaternion_from_euler
```

Comment:
I think all of these changes may already be in `master`. Worth double checking because these are pretty important

Reply:
Don't merge